src/components/organisms/CommentBody/index.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Avatar, Text } from "../../atoms";
+
+const StyledContainer = styled.div`
+  display: inline-flex;
+  padding: 0.8rem;
+  width: 100%;
+`;
+
+const TextWrapper = styled.div`
+  margin-left: 1rem;
+`;
+
+function CommentBody({ commets }) {
+  return commets.map((comment) => {
+    const { nickname, profileImage } = comment.author;
+    return (
+      <StyledContainer key={nickname}>
+        <Avatar url={profileImage} size="sm" />
+        <TextWrapper>
+          <Text text={nickname} size="lg" weight="700" />
+          <Text text={comment.comments} size="base" weight="400" />
+        </TextWrapper>
+      </StyledContainer>
+    );
+  });
```

Comment:
props로 넘어온 commets가 만약 null이라면 에러가 발생할 것 같습니다~!
그럴 가능성은 매우 적겠지만 사전에 미리 처리를 해두면 좋을 것 같습니다😃

하나의 스타일로 통일하는 것이 좋을 것 같은데 25번째 줄에서 comments만 구조분해할당을 통해 가져오지 않고 dot으로 접근한 이유가 따로 있을까요?
```suggestion
  return 조건 추가 && commets.map((comment) => {
    const { comments, author : { nickname, profileImage } } = comment;
    // return 전에 개행이 있으면 가독성이 좋을 것 같습니다.
    return (
      <StyledContainer key={nickname}>
        <Avatar url={profileImage} size="sm" />
        <TextWrapper>
          <Text text={nickname} size="lg" weight="700" />
          <Text text={comments} size="base" weight="400" />
        </TextWrapper>
      </StyledContainer>
    );
  });
```

Reply:
넵 제안해주신 내용으로 수정 하겠습니다